### PR TITLE
Add license and javadoc to Converters

### PR DESCRIPTION
**Pull request by cescoffier**
Add license header to the converter template
Compute the Converter class javadoc

### CODE REVIEW

File: src/main/generated/io/vertx/core/http/HttpClientOptionsConverter.java [hunk 1]
```diff
@@ -1,8 +1,32 @@
...
 
 import io.vertx.core.json.JsonObject;
 import io.vertx.core.json.JsonArray;
 
+/**
+ * Converter for {@link io.vertx.core.http.HttpClientOptions}.
+ *
+ * NOTE: This class has been automatically generated from the {@link io.vertx.core.http.HttpClientOptions} original class using Vert.x codegen.
+ *
+ * @author <a href="http://tfox.org">Tim Fox</a>
```
Comment (purplefox) [inline src/main/generated/io/vertx/core/http/HttpClientOptionsConverter.java]: The author shouldn't be me.

File: src/main/generated/io/vertx/core/metrics/MetricsOptionsConverter.java [hunk 2]
```diff
@@ -1,8 +1,32 @@
...
 
 import io.vertx.core.json.JsonObject;
 import io.vertx.core.json.JsonArray;
 
+/**
+ * Converter for {@link io.vertx.core.metrics.MetricsOptions}.
+ *
+ * NOTE: This class has been automatically generated from the {@link io.vertx.core.metrics.MetricsOptions} original class using Vert.x codegen.
+ *
+ * @author <a href="mailto:julien@julienviet.com">Julien Viet</a>
```
Comment (purplefox) [inline src/main/generated/io/vertx/core/metrics/MetricsOptionsConverter.java]: Or julien

Comment (cescoffier) [inline src/main/generated/io/vertx/core/metrics/MetricsOptionsConverter.java]: It gets the author from the original class.
We can remove the `@author` tag.

**Comment by cescoffier** [comment]
I've removed the tag copy form the original class, so @author won't be inherited anymore.

**Pull request merged**